sparqlbuilder/src/test/java/org/eclipse/rdf4j/sparqlbuilder/examples/BaseExamples.java

```diff
@@ -39,6 +41,8 @@
 	protected static final String DC_NS = "http://purl.org/dc/elements/1.1/";
```

Comment:
I'd rather use the DC.NAMESPACE and FOAF.NAMESPACE to avoid any mistakes, both are already defined in org/eclipse/rdf4j/model/vocabulary package